docs/setup.rst

```diff
@@ -79,6 +79,44 @@ Docker Compose
 Now proceed to `"Setup" <#setup>`__.
 
 
+Heroku
+------
```

Comment:
Worth adding a warning that currently they will need to figure out how to do migrations between upgrades on their own.